src/modules/simulator/accelsim/accelsim.cpp

```diff
@@ -357,23 +353,7 @@ ACCELSIM::init()
 	_mag_reports->get(&mrp);
 
 	/* measurement will have generated a report, publish */
-	_mag->_mag_topic = orb_advertise_multi(ORB_ID(sensor_mag), &mrp,
-					       &_mag->_mag_orb_class_instance, ORB_PRIO_LOW);
-
-	if (_mag->_mag_topic == nullptr) {
-		PX4_WARN("ADVERT ERR");
-	}
-
-	/* advertise sensor topic, measure manually to initialize valid report */
-	_accel_reports->get(&arp);
-
-	/* measurement will have generated a report, publish */
-	_accel_topic = orb_advertise_multi(ORB_ID(sensor_accel), &arp,
-					   &_accel_orb_class_instance, ORB_PRIO_DEFAULT);
-
-	if (_accel_topic == nullptr) {
-		PX4_WARN("ADVERT ERR");
-	}
+	_mag->_mag_topic = orb_advertise_multi(ORB_ID(sensor_mag), &mrp, &_mag->_mag_orb_class_instance, ORB_PRIO_LOW);
```

Comment:
Why do you use `orb_advertise_multi` here instead of relying on `orb_publish_auto`?

Reply:
Great catch!  Looks like it was just leftover from existing.  I've done some additional cleanup in this method as well and pushed up the commit. Thank you!